docs/source/tabular/usage_guides/dataset_object.rst

```diff
@@ -147,8 +147,8 @@ Inferring Categorical Features
 
 .. warning::
     It is highly recommended to explicitly state the categorical features or define their column type to be ``category``.
-    Otherwise, the inherent limitations of the automatic, and may cause inconsistencies (misdetection, different detection between
-    train and test, etc.), and required tuning and adaptions.
+    Otherwise, the inherent limitations of the automatic may cause inconsistencies (misdetection, different detection between
```

Comment:
```suggestion
    Otherwise, the inherent limitations of the automatic type detection may cause inconsistencies (misdetection, different detection between
```

Reply:
Thanks for this